spec/controllers/concerns/users/time_zone_spec.rb

```diff
@@ -37,4 +39,42 @@ class MockController < ApplicationController
       end
     end
   end
+
+  describe "#to_user_timezone" do
+    context "when invalid param is sent" do
+      it "returns the empty string for nil param" do
+        expect(controller.to_user_timezone(nil)).to eq("")
+      end
+
+      it "returns empty string if empty string param provided" do
+        expect(controller.to_user_timezone("")).to eq("")
+      end
+
+      it "returns nil for invalid date string" do
+        expect(controller.to_user_timezone("invalid-date")).to eq(nil)
+      end
+    end
+  end
```

Comment:
Is there a reason that you didn't test the happy path here? getting a time and then returning a value and seeing that the time zone conversion happened?

Reply:
Good point, I will add a test.